app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -53,6 +53,13 @@ export function addComposerUploadHandler(extensions, method) {
   });
 }
 
+const uploadProcessorQueue = [];
+const uploadProcessorActions = {};
+export function addComposerUploadProcessor(queueItem, actionItem) {
```

Comment:
We should add a `cleanUp` that we can call between tests. Otherwise the contents of those arrays might stick around.

Reply:
Maybe I should push this cleanUp stuff in another PR

Reply:
Usually I put them here, but another PR is fine:

https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/tests/helpers/qunit-helpers.js#L247-L266